stdlib/either.mli

```diff
@@ -0,0 +1,88 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*         Gabriel Scherer, projet Parsial, INRIA Saclay                  *)
+(*                                                                        *)
+(*   Copyright 2019 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** Either type.
+
+    Either is the simplest and most generic sum type. It is isomorphic
+    to the [result] type; it should be used when there is no implied
+    success/failure semantics in the choice of one of the constructors.
+
+    @since 4.11 *)
```

Comment:
Why `4.11` here and `NEXT_RELEASE` in `list.mli`?

Reply:
They were written at different times. I think it's fine to ignore this for now.